### DISCUSSION TITLE
Is 6G reasonable RAM usage while doing large git pulls in WSL

### DISCUSSION THREAD

**Discussion by fqrley** [sakai135/wsl-vpnkit#162] [General]
Is 6G reasonable RAM usage while doing large git pulls in WSL.  Every few minutes the CPU usage of wsl-gvproxy.exe will spike to 50-100% as well.  Is this expected behavior?

**Comment by sakai135** [comment]
@fqrley Can you try the version in this PR https://github.com/sakai135/wsl-vpnkit/pull/164? On my machine, the memory usage of wsl-gvproxy.exe stayed at around 20MB with the change while cloning the Linux kernel repo.

**Comment by fqrley** [comment]
Yes, this does seem to be an improvement. I’ve been able to run a large yocto build with this version of vpnkit. Thank you very much. 

Regards,
Matt Farley
